test/TestHelper.cpp

```diff
@@ -234,7 +234,7 @@ void ImportTest::importState(json_spirit::mObject& _o, State& _state)
 	}
 }
 
-void ImportTest::importTransaction(json_spirit::mObject& _o)
+void ImportTest::importTransaction (json_spirit::mObject& _o, eth::Transaction& o_tr)
```

Comment:
I follow the style of other static functions
static void importState(json_spirit::mObject& _o, eth::State& _state);